src/inspector/view/qml/MuseScore/Inspector/general/appearance/internal/VerticalSpacingSection.qml

```diff
@@ -19,27 +19,38 @@
  * You should have received a copy of the GNU General Public License
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
-import QtQuick 2.9
-import MuseScore.Inspector 1.0
-import MuseScore.UiComponents 1.0
+import QtQuick 2.15
 import MuseScore.Ui 1.0
+import MuseScore.UiComponents 1.0
+import MuseScore.Inspector 1.0
 import "../../../common"
 
 InspectorPropertyView {
     id: root
 
     property QtObject minimumDistance: undefined
 
+    property NavigationPanel navigationPanel: null
+    property int navigationRowOffset: 0
```

Comment:
how do we set navigationRowOffset? Is it always 0 or not?

Reply:
yes, we setting it 
https://github.com/musescore/MuseScore/pull/8015/files#diff-d21e999f7e896c07228450ed7a42657cf9623c69aebe4309bab921747c5de1a0R63